EventsExpress.Core/Services/LocationService.cs

```diff
@@ -41,23 +41,35 @@ public async Task<Guid> Create(LocationDto locationDTO)
             return result.Id;
         }
 
-        public LocationDto LocationByPoint(Point point)
+        public Guid? LocationByPoint(Point point)
```

Comment:
Propose to remove this from public interface and change to private method, since this is realization details